docs/int/quickstart/advanced/quickstart-split.md

```diff
@@ -11,6 +11,8 @@ Charon is in a beta state and should be used with caution according to its [Term
 This process should only be used if you want to split an *existing validator private key* into multiple private key shares for use in a Distributed Validator Cluster. If your existing validator is not properly shut down before the Distributed Validator starts, your validator may be slashed.
 
 If you are starting a new validator, you should follow a [quickstart guide](../index.md) instead.
+
+If you use MEV-Boost, make sure you turned off your MEV-Boost service for the time of splitting the keys, otherwise you may hit [the issue](https://github.com/ObolNetwork/charon/issues/2770).
```

Comment:
```suggestion
If you use MEV-Boost, make sure you turned off your MEV-Boost service for the time of splitting the keys, otherwise you may hit [this issue](https://github.com/ObolNetwork/charon/issues/2770).
```